extension/src/popup/components/WarningMessages/index.tsx

```diff
@@ -374,7 +374,12 @@ export const ScamAssetWarning = ({
     setIsSubmitting(false);
   };
 
-  return (
+  return isTrustlineErrorShowing ? (
+    createPortal(
+      <TrustlineError handleClose={() => closeOverlay()} />,
+      document.querySelector("#modal-root")!,
+    )
+  ) : (
```

Comment:
the changes in this file replace the warning modal with a different error modal on tx submission fail